Makefile

```
@@ -0,0 +1,24 @@
+.PHONY: generate clean regenerate
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)